ferveo-python/examples/server_api.py

```diff
@@ -70,9 +66,9 @@
         validators=validators,
         me=validator,
     )
-    aggregate = dkg.aggregate_transcripts(messages)
-    assert pvss_aggregated.validate(dkg)
-    decryption_share = aggregate.create_decryption_share(
+    # assume aggregated transcript obtained through deserialization from side-channel
```

Comment:
Yes, at this point the participant either 1) received transcripts and aggregated them, or 2) downloaded an aggregated transcript from a side-channel. Both are fine.